clients/js/src/utils.ts

```diff
@@ -64,3 +64,18 @@ export async function handleSuccess(response: Response | string | Count200Respon
             return repack(response);
     }
 }
+
+/**
+ * Dynamically imports a specified module, providing a workaround for browser environments.
+ * This function is necessary because we dynamically import optional dependencies
+ * which can cause issues with bundlers that detect the import and throw an error
+ * on build time when the dependency is not installed.
+ * Using this workaround, the dynamic import is only evaluated on runtime
+ * where we work with try-catch when importing optional dependencies.
+ *
+ * @param {string} moduleName - Specifies the module to import.
+ * @returns {Promise<any>} Returns a Promise that resolves to the imported module.
+ */
+export async function importOptionalModule(moduleName: string) {
+  return Function(`return import("${moduleName}}")`)();
```

Comment:
double `}}`?

Reply:
This is a Typo. Good Catch! Should only be one.